File: goormlevel/175011/1. 0 채우기/0 채우기.c
#include <stdio.h>

int main() {
	int arr[100][100];
	int row[100] = {0};
	int col[100] = {0};

	int n = 0;
	if (scanf("%d", &n) != 1) return 0;

	for (int i = 0; i < n; i++) {
		for (int j = 0; j < n; j++) {
			if (scanf("%d", &arr[i][j]) != 1) return 0;

			row[i] += arr[i][j];
			col[j] += arr[i][j];
		}
	}

	int answer = 0;
	for (int i = 0; i < n; i++) {
		for (int j = 0; j < n; j++) {
			if (arr[i][j] == 0) {
					answer += row[i] + col[j];
			}
		}
	}

	printf("%d\n", answer);

	return 0;
}

File: goormlevel/175011/1. 0 채우기/0 채우기.py
n = int(input())
nlist = [list(map(int, input().split())) for _ in range(n)]
row = [sum(row) for row in nlist]
col = [sum(col) for col in zip(*nlist)]

answer = 0
for i in range(n):
	for j in range(n):
		if nlist[i][j] == 0:
				answer += row[i] + col[j]

print(answer)